cypress/integration/gift_cards.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {HomePo} from "../support/home.po";
+import {GiftCardsPo} from "../support/giftCards.po";
+
+describe('Gift cards suite', () => {
+    const homePage = new HomePo();
+    const giftCardPage = new GiftCardsPo();
+
+    beforeEach(() => {
+
+        cy.visit('');
+    })
+
+    it('should ', () => {
+        const price = '100';
+        const giftCardId = '[href*="/B07TMMZKBJ"]';
+
+        homePage.clickOnElem(homePage.giftCardLink);
+        giftCardPage.isCurrentURLContain(giftCardPage.pageUrl);
+        for (let elem of giftCardPage.giftCardLinks) {
+            giftCardPage.isElemVisible(elem);
+        }
+        giftCardPage.clickOnElem(giftCardPage.giftCardLinks[0]);
```

Comment:
you skipped this check: 5. Then user redirected to page with product route
something like this:
        cy.url().should('include', '/product/');